svd/patches/_rename_registers.yaml

```diff
@@ -23,6 +23,11 @@ GPIO:
   _strip:
     - GPIO_
 
+IO_MUX:
```

Comment:
I guess this is uneeded now, as you have already removed the prefix

Reply:
Presumably so!